gnu toolchain version bump

jopadan opened the pull request:
- set versions in build/configs/versions.mk
- add gcc prerequesites updated urls and checksums
- update works with LLVM_VERSION set to -13 and llvm-13 toolchain installed

sleepy-monax commented on build/toolchain/gnu/build.sh:
```diff
@@ -72,6 +74,7 @@ pushd tarballs
         echo "Download gcc prerequisites..."
 
         pushd $GCC_DIRECTORY
+	    patch -Np1 -i ../../prerequisites.patch
```
I don't see the point of this patch? What are the benefits?

jopadan replied:
Latest release versions of dependency math libraries which are not available from the standard mirror path.

jopadan commented on makefile:
```diff
@@ -17,6 +17,13 @@ include sources/build/boards/$(ARCH)-$(BOARD)/build.mk
 
 export LC_ALL=C
 
+ifeq '$(TOOLCHAIN)' 'gnu'
```
I just turned them into warning to be able to compile the code.
I don't see why returning a 64 bit integer causes the oob right now.
The error is not introduced by the changes of the commit.
It might be possible that the new gcc version and the compiler flags are causing it.

Commit:
gnu toolchain version bump

- set versions in build/configs/versions.mk
- add gcc prerequesites updated urls and checksums

Commit:
Merge branch 'brutal-org:main' into main

Commit:
filter-out WARN_CFLAGS from HOST_CFLAGS

Commit:
reset LLVM_VERSION to -13

Commit:
Merge branch 'brutal-org:main' into main

Commit:
Set LLVM_VERSION=-16 after nullptr removal

Commit:
update LLVM and GCC version
